cppapi/client/devapi_attr.tpp

```diff
@@ -397,7 +408,7 @@ void DeviceAttribute::insert(vector<T> &_datum,int _x,int _y)
 }
 
 template <typename T>
-bool DeviceAttribute::template_type_check(T &_datum)
+bool DeviceAttribute::template_type_check(T &TANGO_UNUSED(_datum))
```

Comment:
It is better to remove unused parameters from the code. In CLion use CTRL+F6 when cursor is on the method name to refactor method's signature.

Reply:
In this case, the parameter is actually useful in a way because his type is used in the method.